### PR TITLE
guide for CodiMD

### LINKED ISSUE #157: Guide for HackMD
Adapt https://w.h8.lv/s/how_to_install_hackmd_on_uberspace

### PR DESCRIPTION
**Pull request by Stunkymonkey**
i hope i did everything as expected.
would fix #157

### CODE REVIEW

**Comment by nichtmax** [comment]
First impression: great! Thanks! I’ll take a closer look by the end of the week.

File: source/guide_codimd.rst [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+Again, don't forget to fill in your port number in the last line!
+
+And you're done!
+
+Updates
+=======
+
+.. warning:: 
+  I have not actually tested this so far!
```
Comment (nichtmax) [inline source/guide_codimd.rst]: Hahaha then don't mention it at all ☺️ Keep in mind that we need to answer questions for these guides in support. We can't say "haven't tested it".

File: source/guide_codimd.rst [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+
+If you want to update your CodiMD installation, follow `these instruction <https://github.com/hackmdio/codimd#upgrade>`_, and merge it with this manual. Good luck.
```
Comment (nichtmax) [inline source/guide_codimd.rst]: Imagine trying to follow this guide, something does not work and then there's a "good luck" somewhere in the text. I would be pissed tbh.

**Comment by nichtmax** [comment]
Can you please rebase on our master branch? The CI will then generate a preview from your branch which makes it much easier for us to test your guide.

**Comment by Stunkymonkey** [comment]
@nichtmax sorry i was on holiday
what is up with #218 ?

**Comment by nichtmax** [comment]
@Stunkymonkey We can't get it working, I merged your PR to `Uberspace:codimd` so we can change some things by ourselves but we didn't have any success unfortunately...

**Pull request merged**